task-impls/src/events.rs

```diff
@@ -20,17 +19,19 @@ pub enum SequencingHotShotEvent<TYPES: NodeType, I: NodeImplementation<TYPES>> {
     DAProposalRecv(Proposal<DAProposal<TYPES>>, TYPES::SignatureKey),
     DAVoteRecv(DAVote<TYPES>),
     DACRecv(DACertificate<TYPES>),
-    ViewSyncMessageRecv(ViewSyncMessageType<TYPES>),
-    QuorumProposalSend(Proposal<QuorumProposalType<TYPES, I>>),
+    QuorumProposalSend(Proposal<QuorumProposalType<TYPES, I>>, TYPES::SignatureKey),
     QuorumVoteSend(QuorumVote<TYPES, I::Leaf>),
     DAProposalSend(Proposal<DAProposal<TYPES>>, TYPES::SignatureKey),
     DAVoteSend(DAVote<TYPES>),
-    QCSend(QuorumCertificate<TYPES, I::Leaf>),
+    QCFormed(QuorumCertificate<TYPES, I::Leaf>),
```

Comment:
Nit: I feel `QCSend` is a more consistent name since we have `DACSend` and `ViewSyncCertificateSend`?

Reply:
I changed it to send since this is not he actual send on the network, it's just the event for the main consensus task to know that enough votes were collected to make a QC and we can now propose.